src/graphics/program-lib/program-lib.js

```diff
@@ -1,19 +1,28 @@
 pc.programlib = {
     gammaCode: function (value) {
-        return value===pc.GAMMA_NONE? pc.shaderChunks.gamma1_0PS :
-              (value===pc.GAMMA_SRGBFAST? pc.shaderChunks.gamma2_2FastPS : pc.shaderChunks.gamma2_2PS);
+
+        if (value === pc.GAMMA_SRGB) {
+            return pc.shaderChunks.gamma2_2PS;
+        } else if (value === pc.GAMMA_SRGBFAST) {
+            return pc.shaderChunks.gamma2_2FastPS;
+        } else if (value === pc.GAMMA_SRGBHDR) {
+            return "#define HDR\n" + pc.shaderChunks.gamma2_2PS;
+        } else if (value === pc.GAMMA_SRGBFASTHDR) {
```

Comment:
We've spoke to remove `pc.GAMMA_SRGBFAST` as it is very tiny benefit for only old hardware.
Does it worth adding `pc.GAMMA_SRGBFASTHDR` as well?